File: app/src/test/java/com/tasktree/TreeTaskTest.java
package com.tasktree;

import org.junit.Test;

import static org.junit.Assert.assertNotNull;
import static org.junit.Assert.assertTrue;


public class TreeTaskTest {


    @Test
    public void canCreateAnInstance( ) {
        TreeTask treeTask = new TreeTask();
    }

    @Test
    public void canCreateInstanceWithInitialState( ) {

        TreeTask treeTask = TreeTask.initialState();

        assertNotNull( treeTask );

        assertTrue( treeTask.isInitialState() );
    }

    @Test
    public void getAllPegs() {


    }




}

File: app/src/test/kotlin/com/tasktree/TreeTaskTest.kt
package com.tasktree

import org.junit.Test

import org.junit.Assert.assertNotNull
import org.junit.Assert.assertTrue


class TreeTaskTest {


    @Test
    fun canCreateAnInstance() {
        val treeTask = TreeTask()
    }

    @Test
    fun canCreateInstanceWithInitialState() {

        val treeTask = TreeTask.initialState()

        assertNotNull(treeTask)

        assertTrue(treeTask.isInitialState)
    }

    @Test
    fun getAllPegs() {


    }


}